packages/fields/src/fields/parent/index.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import type { Field } from '@wordpress/dataviews';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+import { __ } from '@wordpress/i18n';
+import { ParentEdit } from './parent-edit';
+import { ParentView } from './parent-view';
+
+const parentField: Field< BasePost > = {
+	id: 'parent',
+	type: 'text',
+	label: __( 'Parent' ),
+	getValue: ( { item } ) => item.parent,
+	Edit: ParentEdit,
+	render: ParentView,
+	enableSorting: false,
```

Comment:
Why disabling sorting? The [endpoint](https://developer.wordpress.org/rest-api/reference/pages/#list-pages) we take the data from supports sorting by `parent`.

Reply:
Fixed with c7f30b90584c96ae728581f14bdcbdd1aa75865c